saq/queue/redis.py

```diff
@@ -340,13 +340,8 @@ async def abort(self, job: Job, error: str, ttl: float = 5) -> None:
                 await self.redis.lrem(self._active, 0, job.id)
 
     async def retry(self, job: Job, error: str | None) -> None:
+        await super().retry(job, error)
```

Comment:
can we make it so you never have to call super? and then the subclasses implement _retry() or something